iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi.hpp

```diff
@@ -129,6 +132,7 @@ class RouDi
   private:
     std::thread m_monitoringAndDiscoveryThread;
     std::thread m_handleRuntimeMessageThread;
+    std::thread listen_thread_watchdog; // 8
```

Comment:
Please prefix members with `m_`

Reply:
@elBoberido I'm sorry for my mistake, it's been fixed

Reply:
No Problem. But do not use ifdefs in the code. Please write an abstraction, similar to what we did IpcChannelType and use that abstraction. For platforms which do not support systemd, write an alternative which basically does nothing when the methods are called. Having ifdefs all over the place makes it hard to maintain and therefore we choose the other approach.

Reply:
@elBoberido I'm not sure if I understood you correctly, please look at the roudi.hpp file did you talk about this? The announcement and the definition will share this is a test for now, thank you :)